assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -18,7 +19,11 @@ const store = createStore(reducer, 'PAPER+DIGITAL');
 
 const content = (
   <Provider store={store}>
-    <Bundles />
+    <div>
+      <Bundles />
+      <br style={{ clear: 'both' }} />
```

Comment:
Yes, we will define a formal way of separating sections of the page. Probably wrapping each section in `<section>` tags.